connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -539,6 +539,92 @@ object functions {
   def grouping_id(colName: String, colNames: String*): Column =
     grouping_id((Seq(colName) ++ colNames).map(n => Column(n)): _*)
 
+  /**
+   * Aggregate function: returns the compact binary representation of the Datasketches HllSketch
```

Comment:
Here and below: I believe we are now returning the updatable sketch representation.